src/components/TimelineComponent.tsx

```diff
@@ -447,7 +479,7 @@ export default class TimelineComponent
         .attr('x', 0)
         .attr('y', 0);
 
```

Comment:
I see 'g' used often as an argument and do not see if defined anywhere. It is not an intuitive variable name so a comment defining it somewhere (if I haven't missed that) would be useful.

Reply:
Can you clarify? This comment doesn’t have a ‘g’ variable.

Reply:
'g' is an SVG group element, it is used to group multiple SVG elements together so that they can all be manipulated at the same time, much like an HTML <div>